kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/Utils.java

```diff
@@ -358,6 +358,7 @@ public static String getPluralFromKind(String kind) {
         break;
       case "NetworkPolicy":
       case "PodSecurityPolicy":
+      case "ServiceEntry": // an Istio resource. Needed as getPluralFromKind is currently not configurable #2489
```

Comment:
Hi, Shall we let this in for now or shall we handle it separately while fixing https://github.com/fabric8io/kubernetes-client/issues/2489